app/src/main/java/com/github/onlynotesswent/model/folder/FolderViewModel.kt

```diff
@@ -255,6 +264,15 @@ class FolderViewModel(private val repository: FolderRepository) : ViewModel() {
         onFailure = onFailure)
   }
 
+  fun getSubFoldersOfNoStateUpdate(
```

Comment:
Consider adding the corresponding documentation for this function

Reply:
Yes good point I will do that.